### PR TITLE
add 'or' condition

### PR DESCRIPTION
**Pull request by Mariusthvdb**
to the conditional cards documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_dashboards/conditional.markdown [hunk 1]
```diff
@@ -12,7 +12,9 @@ The conditional card displays another card based on conditions.
   Screenshot of the conditional card.
 </p>
 
-Note: if there are multiple conditions there will be treated as an 'and' condition. This means that for the card to show, _all_ conditions must be met.
+Note: if there are multiple conditions there will be treated as an 'and' condition by default. This means that for the card to show, _all_ conditions must be met.
```
Comment (c0ffeeca7) [inline source/_dashboards/conditional.markdown]:
```suggestion
Note: if there are multiple conditions, they will be treated as an 'and' condition by default. This means that for the card to show, _all_ conditions must be met.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Mariusthvdb :+1:

**Comment by c0ffeeca7** [comment]
going to close this as a duplicate of #30593 .

**Comment by Mariusthvdb** [comment]
got to be honest, and see the other was a duplicate of this.

It even used my exact yaml, which is now in the documentation..

anyways, 'or' is documented..

**Comment by c0ffeeca7** [comment]
yes, you are right. I should have reviewed this one first. Thank you for creating and providing this example.

**Pull request closed without merging**